lib/mapper.js

```diff
@@ -164,14 +167,16 @@ function createBaseTrackProperties(track, settings) {
   if (settings.prefixProperties) {
     properties = prefix(track.event(), properties);
   }
-
-  return extend(
-    properties,
-    {
+  // Only apply userAgent property on `visited site` event
+  if (track.event() === 'visited site') {
```

Comment:
We should pass in `_ua` for all events, but we should not pass in empty/undefined user agents.